testapp/src/main/java/com/google/android/agera/testapp/NotesActivity.java

```diff
@@ -55,7 +53,7 @@
   private static final Executor networkExecutor = newSingleThreadExecutor();
   private static final Executor calculationExecutor = newSingleThreadExecutor();
 
-  private RepositoryAdapter adapter;
+  private Adapter adapter;
```

Comment:
Wouldn't this cause the "raw type" warning in some cases? (Android Studio is pretty lax when it comes to using raw types, so you don't see it).